ext/standard/filestat.c

```diff
@@ -765,6 +765,10 @@ PHPAPI void php_stat(const char *filename, size_t filename_length, int type, zva
 		"dev", "ino", "mode", "nlink", "uid", "gid", "rdev",
 		"size", "atime", "mtime", "ctime", "blksize", "blocks"
 	};
+	zval *stat_sb_addresses[13] = {
```

Comment:
It should be possible to leave off the number of elements both here and on stat_sb_names.